src/util/cli.js

```diff
@@ -40,6 +40,11 @@ exports.serverArgs = {
     type: "boolean",
     default: false
   },
+  passwordFile: {
+    describe: "Use a password from a file to authenticate",
```

Comment:
Maybe better to use "Read" instead of "Use"? Makes it clearer it's just loading the file with no special parsing, I think.